headlamp-k8s/headlamp
# Error boundary in routes

This PR adds an error boundary to the router components + adds info about the error to the error page.

![New error boundary](https://github.com/user-attachments/assets/dc557dc3-68cf-48b4-af2d-115c3184cd2a)
![New error boundary with errors](https://github.com/user-attachments/assets/4b7f6952-8b34-4102-b33e-d6a7aa23c340)

## How to reproduce
1. Change the members in a table that a list view is rendering (for example, in the Job list) to access a non-existing (e.g. `item.somethingThatDoesntExist`);
2. Verify that the error page is displayed just in the main area of Headlamp and not the entire page

## Done
- [X] Error boundary for route
- [X] Error details in error page
- [x] Story for Storybook
